commit cbffffbd7b2f314fbfa593a6da10febf279313cb
Author: Ralph Little <skelband@gmail.com>
Date:   Mon Aug 22 20:55:59 2022 -0700

    epsonds: Added new model XP-2200 series.
    
    Patch supplied by Nakamura Iwao from Epson Japan.

diff --git backend/epsonds.c backend/epsonds.c
index 5fac5d41e..72d01d8a7 100644
--- a/backend/epsonds.c
+++ b/backend/epsonds.c
@@ -1258,6 +1258,7 @@ const  epsonds_profile_map epsonds_models_predefined[] = {
   {0x118A, "PID 118A","ET-2810 Series", 7},
   {0x118A, "PID 118A","L3250 Series", 7},
   {0x119B, "PID 119B","XP-2150 Series", 7},
+  {0x11B1, "PID 11B1","XP-2200 Series", 7},
   {0x00, "","", 0x00 }
 };
 
diff --git doc/descriptions/epsonds.desc doc/descriptions/epsonds.desc
index 294ab5ae7..989caf88f 100644
--- a/doc/descriptions/epsonds.desc
+++ b/doc/descriptions/epsonds.desc
@@ -353,3 +353,8 @@
 :interface  "USB"
 :usbid      "0x04b8" "0x119b"
 :status     :complete
+
+:model      "XP-2200 Series"
+:interface  "USB"
+:usbid      "0x04b8" "0x11b1"
+:status     :complete
